celestia/celestia.go

```diff
@@ -74,7 +74,7 @@ func (c *CelestiaDA) Commit(daBlobs []da.Blob) ([]da.Commitment, error) {
 }
 
 // Submit submits the Blobs to Data Availability layer.
-func (c *CelestiaDA) Submit(daBlobs []da.Blob) ([]da.ID, []da.Proof, error) {
+func (c *CelestiaDA) Submit(daBlobs [][]byte, options *da.SubmitOptions) ([][]byte, [][]byte, error) {
```

Comment:
The mock `Submit` calls will need to be updated as well.

Reply:
I just fixed it